package.json

```diff
@@ -11,6 +11,7 @@
     "babel-loader": "7.1.1",
     "babel-preset-react-app": "^3.0.3",
     "babel-runtime": "6.26.0",
+    "bootstrap": "4.0.0-beta",
```

Comment:
Could you describe in the PR how adding bootstrap/reactstrap is useful? (is it to make the sidebar prettier?)